src/wizard/javascript/svelte/with-error-monitoring-and-performance.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Svelte
+doc_link: https://docs.sentry.io/platforms/javascript/guides/svelte/
+support_level: production
+type: framework
+---
+
+## Install
+Sentry captures data by using an SDK within your application’s runtime.
+
+```bash
+# Using yarn
+yarn add @sentry/svelte
+
+# Using npm
+npm install --save @sentry/svelte
+```
+
+## Configure
+Initialize Sentry as early as possible in your application's lifecycle.
```

Comment:
This used to be missing a hint that it should be placed inside `main.ts/js` - is that on purpose?
Also noticed this in a few other files.
https://github.com/getsentry/sentry-docs/blob/1ad5d4885509a6818a8777bcd5e2d4dc1d8defa0/src/wizard/javascript/svelte.md#L18

Reply:
@Lms24 for Svelte will it always be main or similar? Is the framework strict enough for that?

Reply:
Fixed with bc11960d683d2d879138d3d689f03348c92eb5ce